### DISCUSSION TITLE
Drag and Drop

### DISCUSSION THREAD

**Discussion by Bod1973** [zsarnett/Custom-Grid-View#29] [General]
Hi.  I have followed your video (08/11/21) but when in Lovelace Edit Dashboard there are no changes etc.

**Comment by zsarnett** [comment]
This most likely doesn't work with newer versions... 

I have plans to start this again next year for sure. Planning Q1 of 2022 to be the release. I know its a long time.

**Reply by topche** [reply]
My version is Home Assistant 2021.12.9 and add this , but there is no changes and can't DnD. 
Did I miss something ?

**Reply by stefan-cremi** [reply]
Are you still planning to release it in this quarter? @zsarnett

**Comment by zdavesf** [comment]
Do you have any intent on making this work in HA Core?  This looks amazing...
